libs/hiraya/src/components/Alert/index.tsx

```diff
@@ -0,0 +1,56 @@
+import { ThemeProvider } from '@emotion/react';
+import { AlertStyled, AlertProps } from './style';
+import { CssBaseline, Typography } from '@mui/material';
+import theme from './theme';
+import { AlertTitleStyled } from './style';
+import { Button } from '../Button';
+import IconButton from '@mui/material/IconButton';
+import CloseIcon from '@mui/icons-material/Close';
+
+const handleClick = () => {
+  alert('Alert is Closed');
+};
+
+const Alert: React.FC<AlertProps> = ({
+  color,
+  title,
+  emphasis,
+  message,
+  showButton,
+  showTitle,
+  isDismissible,
+  ...props
+}) => {
+  return (
+    <ThemeProvider theme={theme}>
```

Comment:
I dont think you should put the theme provider or css baseline at component level,

you should make a wrapper and wrap the preview.tsx config on storybook https://storybook.js.org/docs/writing-stories/decorators